### ISSUE TITLE
4 runs with logits and L1 metrics

### ISSUE THREAD

**Issue by janchk** [aifoundry-org/MHAQ#28]

**Comment by janchk** [comment]
![Image](https://github.com/user-attachments/assets/bc95a40d-5aee-4692-95eb-e9946dbd8a66)

**Comment by janchk** [comment]
Not a very satisfied results. Big deviation. Probably because that logits without normalization in L1 distance introduces additional outliers.
